src/components/search/History/SearchHistory.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Icon as UIIcon, List as UIList } from '@faststore/ui'
+import Button from 'src/components/ui/Button'
+import './SearchHistory.scss'
+import Link from 'src/components/ui/Link'
+import Icon from 'src/components/ui/Icon'
+import useSearchHistory from 'src/sdk/search/useSeachHistory'
+import { formatSearchState, initSearchState } from '@faststore/sdk'
+
+const doSearch = (term: string) => {
+  const { pathname, search } = formatSearchState(
+    initSearchState({
+      term,
+      base: '/s',
+    })
+  )
+
+  return `${pathname}${search}`
+}
+
+interface SearchHistoryProps {
+  onClear: () => void
+}
+
+const SearchHistory = ({ onClear }: SearchHistoryProps) => {
+  const { searchHistory } = useSearchHistory()
+
+  return (
+    <section data-store-search-history className="history__section">
+      <div className="history__header">
+        <p className="history__title">History</p>
+        <Button variant="tertiary" onClick={onClear}>
+          Clear
+        </Button>
+      </div>
+      <UIList variant="ordered">
+        {searchHistory.map((item, index) => (
+          <li key={index}>
+            <Link variant="display" to={doSearch(item)}>
+              <div>
```

Comment:
I was going to comment that block elements such as `div`s were not allowed inside inline elements such as `a`s, but it appears [this rule doesn't apply anymore with HTML 5](https://stackoverflow.com/a/1828032/7970105)! _Today I learned_.

Reply:
TIL! 😄